aws/src/main/java/org/apache/iceberg/aws/RESTSigV4Signer.java

```diff
@@ -51,7 +51,11 @@
  * <p>See <a
  * href="https://docs.aws.amazon.com/general/latest/gr/signing-aws-api-requests.html">Signing AWS
  * API requests</a> for details about the protocol.
+ *
```

Comment:
```suggestion
 *   @deprecated since 1.9.0, will be removed in 1.10.0; use {@link RESTSigV4AuthManager} instead.
```

Good to mention the current version of deprecation as it will be easier for the release manager to clean it up during the next release. 
Iceberg has guidelines around it: https://iceberg.apache.org/contribute/?h=contri#deprecation-notices